Gems/OpenXRVk/Code/Source/OpenXRVkSwapChain.cpp

```diff
@@ -16,6 +16,33 @@
 #include <OpenXRVk/OpenXRVkUtils.h>
 #include <XR/XRFactory.h>
 
+// TODO: Expose ConvertFormat in Vulkan RHI::Reflect
+#include <../Source/RHI/Formats.inl>
```

Comment:
This will be addressed in a follow up PR.

Reply:
This has bee done, these are the PRs:
https://github.com/o3de/o3de/pull/11329
https://github.com/o3de/o3de-extras/pull/15